app/src/main/java/fr/free/nrw/commons/profile/leaderboard/UserDetailAdapter.java

```diff
@@ -84,6 +86,17 @@ public void onBindViewHolder(@NonNull UserDetailAdapter.DataViewHolder holder, i
             holder.getContext().getResources().getString(R.string.count_prefix),
             leaderboardResponse.getCategoryCount()));
 
+        avatar.setOnClickListener(new DebouncingOnClickListener() {
```

Comment:
Please add a comment explaining what this block of code does. Thanks!

Reply:
Its generally a good idea to have well documented readable code. 
How about adding "Show instructions to set up avatar" as an additional comment ?